### ISSUE TITLE
No Peacock commands available until changing setup by hand

### ISSUE THREAD

**Issue by jotabe-net** [johnpapa/vscode-peacock#290]
**Versions (please complete the following information):**

- OS: Windows
- VS Code: 1.38.1
- Peacock 3.1.5

**Describe the bug**
When writing "Peacock" in teh command palette, only "Open the documentation" appears.

**To Reproduce**
Install Peacock on a clean instance of Visual Studio Code (show that it doesn't have previous configuration in settings.json)

Open the command palette and type "Peacock"

**Expected behavior**
You should see all Peacock commands, but only gte "Open the Documentation"

**Additional context**
If you modify settings.json and add options like "peacock.affectTitleBar":true, or "peacock.affectTabActiveBorder":true, when you type "Peacock" in the command palette, you get all the Peacock commands.

**Comment by johnpapa** [comment]
Please see the docs. It explains this scenario which is due to not having a workspace open.

**Comment by MatthewRandall** [comment]
Just wanted to add something to this in 2024. I have been using Peacock extensively (great extension!) 

I created a new workspace, opened a file. Saved the workspace. When I went to attempt to set the color the only command option available was "Peacock: Open the Documentation". I checked over the settings. I disabled and re-enabled local and global. Nothing seemed to work. As I was about to give up trying to figure it out and reinstall the extension I realized that the workspace did not have a folder added. I thought to myself, "I wonder if this is the problem as all my other workspaces have at least one folder added to the workspace." 

Bingo. Soon as I added a folder to the workspace the "Peacock: Change to a favorite color" command was available and everything it back to normal. I am not sure if this is a bug or intended behavior just thought I would share my findings and potential fix for any others coming here for the same reason I have this many years later.
